shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -37,6 +39,17 @@
   return blink::SemanticsAction::kScrollUp;
 }
 
+bool GeometryComparator(SemanticsObject* a, SemanticsObject* b) {
+  // Should a go before b?
+  CGRect rectA = [a accessibilityFrame];
+  CGRect rectB = [b accessibilityFrame];
+  CGFloat top = rectA.origin.y - rectB.origin.y;
+  if (top == 0) {
+    return rectA.origin.x - rectB.origin.x < 0;
+  }
+  return top < 0;
```

Comment:
ll. 47-50: Since these are all `CGFloat` comparisons, consider using `0.0` for readability, even though they'll get dealt with through implicit conversion.